utils/mock-data-loader/.eslintrc.js

```diff
@@ -1,4 +1,4 @@
-module.exports = {
+eslintrc.jsmodule.exports = {
```

Comment:
I'm not sure if this one is working as expected?

Reply:
Ah! typo, good spot have fixed.